airflow/ui/src/components/FlexibleForm/NormalRow.tsx

```diff
@@ -0,0 +1,49 @@
+/*!
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import { Field, Stack } from "@chakra-ui/react";
+import Markdown from "react-markdown";
+import remarkGfm from "remark-gfm";
+
+import type { ParamSpec } from "src/queries/useDagParams";
+
+import type { FlexibleFormElementProps } from ".";
+import { SelectElement } from "./SelectElement";
+
+const isRequired = (param: ParamSpec) =>
+  // The field is required if the schema type is defined.
+  // But if the type "null" is included, then the field is not required.
+  // We assume that "null" is only defined if the type is an array.
+  Boolean(param.schema.type) && (!Array.isArray(param.schema.type) || !param.schema.type.includes("null"));
+
+/** Render a normal form row with a field that is auto-selected */
+export const NormalRow = ({ key, name, param }: FlexibleFormElementProps) => (
```

Comment:
```suggestion
export const NormalRow = ({ name, param }: FlexibleFormElementProps) => (
```

Reply:
I think something like `FlexibleField` is more descriptive than `NormalRow`

Reply:
With "Normal Row" I wanted to say that this is "not hidden" - maybe there will be more complex field types int he future... "Normal" for me is a label plus some entry.

We have too many components with "Fields" already... this here renders a row. Would `FormRow` be also OK?